client/src/permissions/UserRoles.js

```diff
@@ -0,0 +1,15 @@
+export const UserRoles = Object.freeze({
+  Admin: "Admin",
+  Host: "Host",
+  Developer: "Developer",
+  Mapsetter: "Mapsetter",
+  AllStarMapSetter: "All-Star Mapsetter",
+  Pooler: "Pooler",
```

Comment:
`Pooler` isn't a role that exists -- currently there's `Mapsetter` and `Head Pooler` which exist (see TourneyStaff.js)

i originally named it "mapsetter" instead of "pooler" because the old gts forum posts called them "mapsetters", but tbh that name sucks and now looks inconsistent with "head pooler". i'd like to rename it at some point in the future.

Reply:
Removed in dceabdcc97e48aecd10dbdb9116d28b74c607d57